frontend/app/craco.config.js

```diff
@@ -13,8 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const CircularDependencyPlugin = require("circular-dependency-plugin")
```

Comment:
Removed because we don't use webpack for the library but we can find a way to add this back in.